dao/src/main/java/org/thingsboard/server/dao/model/sql/WidgetTypeInfoEntity.java

```diff
@@ -58,6 +62,10 @@ public class WidgetTypeInfoEntity extends AbstractWidgetTypeEntity<WidgetTypeInf
     @Column(name = ModelConstants.WIDGET_TYPE_WIDGET_TYPE_PROPERTY)
     private String widgetType;
 
+    @Convert(converter = WidgetBundleEntityInfosConverter.class)
+    @Column(name = ModelConstants.WIDGET_BUNDLES_PROPERTY)
+    private List<EntityInfo> bundles;
```

Comment:
Use JsonNode here and in `toData` - convert to list of EntityInfo. Get rid of WidgetBundleEntityInfosConverter